models/endpoints/endpoints.json

```diff
@@ -14543,13 +14543,21 @@
             "credentialScope" : {
               "region" : "us-gov-east-1"
             },
-            "hostname" : "ec2.us-gov-east-1.amazonaws.com"
+            "hostname" : "ec2.us-gov-east-1.amazonaws.com",
+            "variants" : [ {
+              "hostname" : "ec2.us-gov-east-1.amazonaws.com",
+              "tags" : [ "fips" ]
+            } ]
           },
           "us-gov-west-1" : {
             "credentialScope" : {
               "region" : "us-gov-west-1"
             },
-            "hostname" : "ec2.us-gov-west-1.amazonaws.com"
+            "hostname" : "ec2.us-gov-west-1.amazonaws.com",
+            "variants" : [ {
+              "hostname" : "ec2.us-gov-east-1.amazonaws.com",
```

Comment:
Should be "ec2.us-gov-west-1.amazonaws.com" ?

Reply:
Sorry for the typo..

Reply:
Fixed.